internal/configs/configmaps_test.go

```diff
@@ -342,6 +342,78 @@ func TestParseMGMTConfigMapWarnings(t *testing.T) {
 			},
 			msg: "enforce-initial-report set empty",
 		},
+		{
```

Comment:
can we add one positive and negative for usage-report-endpoint?

Reply:
The positive cases are already in main here, that test is just for negative cases/warnings - https://github.com/nginxinc/kubernetes-ingress/pull/6987/files#diff-cc6c01f505635e7cc22995ac2ef715d4e8929bbc3a3b6258a2d6bb4ccb54b678R466